src/main/java/org/spongepowered/mod/event/SpongeEventHooks.java

```diff
@@ -52,4 +57,23 @@ public void onEntityDeathEvent(LivingDeathEvent event) {
         SpongeHooks.logEntityDeath(event.entity);
     }
 
+    @SideOnly(Side.SERVER)
+    @SubscribeEvent
+    public void onChunkPrePopulate(PopulateChunkEvent.Pre event) {
```

Comment:
Might be best to have ours run first so other mods can come behind and do some tweaking based on our results in the PRE event.